app/views/static/cloud_subnet/cloud-subnet-form.html.haml

```diff
@@ -0,0 +1,130 @@
+                            'disabled'  => true,
+                            'maxlength' => 128,
+                            'ng-model'  => 'vm.cloudSubnetModel.ext_management_system.name'}
+    .form-group{'ng-class' => '{"has-error": angularForm.cloud_tenant_id.$invalid}',
+                'ng-if' => 'vm.cloudSubnetModel.ems_id'}
+      %label.col-md-2.control-label
+        = _('Cloud Tenant Placement')
+      .col-md-8{'ng-if' => 'vm.newRecord'}
+        %select{'name'                        => 'cloud_tenant_id',
+                'ng-model'                    => 'vm.cloudSubnetModel.cloud_tenant.id',
+                'ng-options'                  => 'tenant.id as tenant.name for tenant in vm.available_tenants',
+                'pf-select'                   => true,
+                'required'                    => '',
+                'selectpicker-for-select-tag' => ''}
+          %option{'value' => ''}
+            = "<#{_('Choose')}>"
+        %span.help-block{'ng-show' => "angularForm.cloud_tenant_id.$error.required"}
+          = _('Required')
+      .col-md-8{'ng-if' => '!vm.newRecord'}
+        %input.form-control{'type'      => 'text',
+                            'name'      => ' clout_tenant_name',
+                            'maxlength' => 128,
+                            'ng-model'  => 'vm.cloudSubnetModel.cloud_tenant.name',
+                            'disabled'  => true}
+  %h3
+    = _('Cloud Subnet details')
+  .form-horizontal
+    .form-group{'ng-class' => '{"has-error": angularForm.network_id.$invalid}'}
+      %label.col-md-2.control-label
+        = _('Network')
+      .col-md-8{'ng-if' => 'vm.newRecord'}
+        %select{'name'                        => 'network_id',
+                'ng-model'                    => 'vm.cloudSubnetModel.network_id',
+                'ng-options'                  => 'network.ems_ref as network.name for network in vm.available_networks',
+                'pf-select'                   => true,
+                'required'                    => '',
+                'selectpicker-for-select-tag' => ''}
+          %option{'value' => ''}
+            = "<#{_('Choose')}>"
+        %span.help-block{'ng-show' => 'angularForm.network_id.$error.required'}
+          = _('Required')
+      .col-md-8{'ng-if' => '!vm.newRecord'}
+        %input.form-control{'type'      => 'text',
+                            'name'      => 'network_name',
+                            'maxlength' => 128,
+                            'ng-model'  => 'vm.cloudSubnetModel.cloud_network.name',
+                            'disabled'  => true}
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Subnet Name')
+      .col-md-8
+        %input.form-control{'type'      => 'text',
+                            'name'      => 'name',
+                            'ng-model'  => 'vm.cloudSubnetModel.name',
+                            'maxlength' => 128}
```

Comment:
`Subnet name` should be required. It used to be before [this change](https://github.com/ManageIQ/manageiq-ui-classic/pull/1545/files#diff-2ac4226efb29162218c49f700b6aea34L10)

Reply:
This has to be required for everyone but OpenStack. There's no code letting user know it cannot be saved because of missing `name`.

Reply:
Nevermind. It's not supported by anyone else but OpenStack in MiQ.